vllm-tensorizer/Dockerfile

```diff
@@ -42,8 +49,16 @@ RUN git clone --filter=blob:none --depth 1 --no-single-branch --no-checkout \
 
 FROM builder-base as vllm-builder
 WORKDIR /workspace
-COPY --from=vllm-downloader /git/vllm /workspace
-RUN python3 -m pip wheel -w /wheels -v --no-cache-dir --no-build-isolation --no-deps ./
+ENV LD_LIBRARY_PATH /usr/local/cuda/lib64:${LD_LIBRARY_PATH}
+RUN if [ -d "/usr/local/cuda" ]; then echo "Directory /usr/local/cuda exists."; else exit 1; fi
```

Comment:
1. `/usr/local/cuda/lib64` is added to `ld`'s search path by `/etc/ld.so.conf.d/000_cuda.conf` when you run `ldconfig`, so this does not need to be prepended to `LD_LIBRARY_PATH`
2. This check isn't needed (it will almost certainly pass anyway) as long as the following build succeeds
```suggestion
```